### ISSUE TITLE
Unused method parameter not discovered

### ISSUE THREAD

**Issue by the21st** [HairyFotr/linter#28]
Hi! Thanks for your work on the Linter library!

This method does not raise an unused method parameter by Linter:

```scala
def foo(param1: String, param2: Int) = {
	param2 + 1
}
```

version: linter_2.11-0.1-SNAPSHOT

**Comment by HairyFotr** [comment]
There's a filter for simple methods, so if the method only contains a single expression the warning is not raised. I don't remember the reason for putting it in, so I'll run a version without the filter on a few codebases and commit, if it doesn't cause too many false positives.

Thanks for reporting!

**Comment by HairyFotr** [comment]
Found another linter bug with default parameters along the way, and a bug in one application, so thanks again.
